src/reanimated2/shareables.ts

```diff
@@ -277,8 +274,8 @@ Offending code was: \`${getWorkletCode(value)}\``);
         toAdapt,
         shouldPersistRemote
       );
-      _shareableCache.set(value, adopted);
-      _shareableCache.set(adopted, _shareableFlag);
```

Comment:
The idea was to keep `_shareableCache` "private" (i.e. don't export it) and populate the hashmap only `registerShareableMapping` for the purpose of consistency. Certainly, it would work better if we didn't read directly from `shareableCache` but this sounds like a separate PR to me 😛

Reply:
🤪